buildout.cfg

```diff
@@ -36,6 +36,14 @@ auto-checkout = snovault
 ini-templates = ${buildout:directory}/ini-templates
 es-ip = localhost
 es-port = 9201
+primary_indexer_queue_type = Simple
+primary_indexer_queue_server = true
+primary_indexer_queue_worker = true
+primary_indexer_queue_worker_processes = 16
+primary_indexer_queue_worker_chunk_size = 1024
+primary_indexer_queue_worker_batch_size = 5000
```

Comment:
Simple queue should have a batch_size of 2000000 or greater than the number of objects.